### PR TITLE
[RFC] Remove global variable screen_char_attr in screen.c

### PR DESCRIPTION
**Pull request by mmhl**
Hi, I am starting to look at the vim codebase. What I've found is this global variable, that can be removed easily.
Changed signatures of `screen_char` and `screen_char_2` to get char attribute as a parameter.

### CODE REVIEW

**Comment by justinmk** [comment]
Nice, I hope there are more of these opportunities.

@mmhl I see that you signed the CLA, but you did not add your timestamp, which was emailed, to the [wiki page](https://github.com/neovim/neovim/wiki/CLA-confirmation-page). Would you mind doing so?

**Comment by mmhl** [comment]
Done, thanks.

**Comment by elmart** [comment]
LGTM.

For future work, and other, less isolated globals, please take into account what discussed at https://github.com/neovim/neovim/pull/1389#issuecomment-62317626.

**Comment by Pyrohh** [comment]
Beside the comment, LGTM

File: src/nvim/screen.c [hunk 1]
```diff
@@ -5988,7 +5985,7 @@ void reset_cterm_colors(void)
  * Put character ScreenLines["off"] on the screen at position "row" and "col",
  * using the attributes from ScreenAttrs["off"].
```
Comment (philix) [inline src/nvim/screen.c]: You need to mention `char_attr` in the comments.

**Comment by philix** [comment]
In the current codebase, `screen_char_attr` is never changed. @mmhl can you check `screen_char_attr` can be simply removed?

**Comment by justinmk** [comment]
As mentioned, `screen_char_attr` was neutered by 3baba1e7bc6698e6bc9f1d37fce88b30d6274bc9.

**Pull request closed without merging**